examples/react-app/src/FormElements/Sliders.tsx

```diff
@@ -1,86 +1,124 @@
-import { useState } from 'react';
-import { PlmgSlider } from '@ducky/plumage-react';
+import { PlmgButton, PlmgSlider } from '@ducky/plumage-react';
+import { SLIDERS } from './Constants/Sliders';
 
-export default function Sliders() {
-  const [sliders, setSlider] = useState({
-    stepped: 0,
-    decimal: 0.1,
-    large: 100000,
-    marked: 0,
-  });
-
-  const handleSlider = (event: any) => {
-    const { name } = event.target;
-    const { value } = event.detail;
-
-    if (sliders[name] !== value) {
-      setSlider((prevState) => ({
-        ...prevState,
-        [name]: parseFloat(value),
-      }));
-    }
-    event.preventDefault();
-    event.stopPropagation();
-  };
-
-  const handleSubmit = () => {
-    alert(`Submitted Slider Values
-      Stepped Slider: ${sliders.stepped}
-      Decimal Slider: ${sliders.decimal}
-      Large Value Slider: ${sliders.large}
-    `);
+export type Props = {
+  onUpdate: (newValue: number, label: string) => void;
+  sliders: {
+    [key: string]: number;
   };
+};
 
+export default function SlidersForm(props: Props) {
+  const { sliders } = props;
+  const { onUpdate } = props;
```

Comment:
You can consolidate your destructuring. 
```suggestion
  const { sliders, onUpdate } = props;
```